lib/navigation/control/crosscheck.hpp

```diff
@@ -44,7 +48,12 @@ class Crosscheck {
    * @return true Keyence and wheel encoders agree
    * @return false Keyence and wheel encoders disagree
    */
-  SensorChecks checkEncooderKeyence(const core::EncoderData encoder_data,
-                                    const core::KeyenceData keyence_data);
+  SensorChecks checkEncoderKeyence(const core::Float encoder_displacement,
+                                   const core::Float keyence_displacement);
+
+  core::ILogger &logger_;
+  const core::ITimeSource &time_;
+  static constexpr std::uint8_t kMaxAllowedAccelerometerEncoderDifference = 5;
+  static constexpr std::uint8_t kMaxAllowedKeyenceEncoderDifference       = 12;
```

Comment:
do we have any reasoning for these thresholds? (Just to avoid them feeling like magic)

Reply:
Yeah that would be better. Can't really judge the values lol